tests/Feature/WPSEOTest.php

```diff
@@ -24,21 +31,22 @@ function test_get_term_option() {
 
 	function test_intersect_term_option() {
 		$this->assertCount(
-			3,
-			WP_SEO::instance()->intersect_term_option( array() ),
+			2,
```

Comment:
is this an expected/explained change?

Reply:
This is testing [this function](https://github.com/alleyinteractive/wp-seo/blob/v2.0/php/class-wp-seo.php#L151), the count having changed because we've removed `keywords` from the defaults.